manual/manual/cmds/unified-options.etex

```diff
@@ -375,6 +375,14 @@ each functor application generates new types in its result and
 applying the same functor twice to the same argument yields two
 incompatible structures.
 
+\nat{%
+\item["-no-float-const-prop"]
+Deactivates the constant propagation for floating-point operations.
+With this option no propagation of floating-point constants is
+performed. This option should be given if the program changes the
```

Comment:
The second sentence seems to repeat the meaning of the first one; isn't there a redundancy here?

Reply:
Yes, I  would suggest removing the line.